compiler/passes/scopeResolve.cpp

```diff
@@ -864,14 +873,6 @@ static astlocT* resolveUnresolvedSymExpr(UnresolvedSymExpr* usymExpr,
     resolveUnresolvedSymExpr(usymExpr, sym);
   } else {
     updateMethod(usymExpr);
-
-#ifdef HAVE_LLVM
-    if (nSymbols == 0 && gExternBlockStmts.size() > 0) {
-      Symbol* got = tryCResolve(usymExpr, name);
-      if (got != NULL)
-        resolveUnresolvedSymExpr(usymExpr, got);
-    }
-#endif
```

Comment:
Did this get moved somewhere else?

Reply:
Yes, I've moved it to lookupThisScopeAndUses